pkg/network/tracer/offsetguess/conntrack.go

```diff
@@ -129,7 +129,7 @@ func (c *conntrackOffsetGuesser) checkAndUpdateCurrentOffset(mp *maps.GenericMap
 
 		if c.status.Saddr == expected.saddr {
 			// the reply tuple comes always after the origin tuple
-			c.status.Offset_reply = c.status.Offset_origin + sizeofNfConntrackTuple
+			c.status.Offset_reply = c.status.Offset_origin + sizeofNfConntrackTupleHash
```

Comment:
I read the kernel source linked in your jira comment but I don't fully understand why the offset would be by hash and not the tuple struct?

Reply:
Just by tuple struct would get us to the start of the next hash object (reply); we want to get to the tuple in the hash.